kvbc/tools/sparse_merkle_db/include/sparse_merkle_db_editor.hpp

```diff
@@ -185,14 +185,33 @@ struct GetValue {
   }
 };
 
+struct RemoveMetadata {
+  std::string description() const {
+    return "removeMetadata\n"
+           "Removes metadata and state transfer data from RocksDB";
+  }
+  std::string execute(const v2MerkleTree::DBAdapter &db_adapter, const CommandArguments &) const {
+    const concordUtils::Sliver b{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::BFT),
+                                             static_cast<char>(storage::v2MerkleTree::detail::EBFTSubtype::Metadata)}};
+    const concordUtils::Sliver e{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::Key)}};
```

Comment:
Nit: Maybe we can call them `begin` and `end` to make it clearer.

Reply:
Done